[DOCS] Remove examples of specifying a test_backends list of dicts (#7816)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```

:::note

> The optional `only_for` and `suppress_test_for` keys may be specified at the top-level (next to `data` and `tests`) or within specific tests (next to `title`, etc).

Reviewer: ```suggestion
The optional `only_for` and `suppress_test_for` keys can be specified at the top-level (next to `data` and `tests`) or within specific tests (next to `title`, and so on).
```
